### PR TITLE
Standardise padding and margin utility classes

### PR DESCRIPTION
**Pull request by precious-onyenaucheya-ons**
### What is the context of this PR?

Fixes: https://github.com/ONSdigital/design-team/issues/74

This PR aligns the configuration of the padding with the margin file. The sizes in the padding file have been updated to match those in the margin file. Additionally, !important has been removed from the no-size class. The _pad.scss file has been renamed to _padding.scss, and all references to it have been updated for consistency.

### How to review this PR

1.  Review the tests and visual tests to ensure the padding sizes are correct.

### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

File: src/scss/utilities/_padding.scss [hunk 1]
```diff
@@ -1,11 +1,12 @@
 // e.g ons-u-pl-m
 $sizes: (
-    no: 0 !important,
```
Comment (rmccar) [inline src/scss/utilities/_padding.scss:3]: !important should be added on line 22 in the css file like in the margin one. Other than that looks good!

Comment (precious-onyenaucheya-ons) [reply src/scss/utilities/_padding.scss:3]: done

File: src/scss/utilities/_padding.scss [hunk 2]
```diff
@@ -1,11 +1,12 @@
 // e.g ons-u-pl-m
```
Comment (rmccar) [inline src/scss/utilities/_padding.scss:1]: Think we can remove this line too, after that it looks good

Comment (precious-onyenaucheya-ons) [reply src/scss/utilities/_padding.scss]: removed

**Review by rmccar — APPROVED** [review]

**Review by alessioventuriniAND — APPROVED** [review]

**Review by SriHV — APPROVED** [review]

**Review by SriHV — APPROVED** [review]

**Review by alessioventuriniAND — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Pull request merged**